### PR TITLE
Fixed issues "Research Page Card Modification: Fonts"

### PR DESCRIPTION
**Pull request by RitikPandey1**
**Proposed Changes**
Title: Semi Bold
Authors: Medium
Conference: Regular
Issue #423
for font weight values reference : https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

### CODE REVIEW

File: assets/css/research.css [hunk 1]
```diff
@@ -223,7 +228,7 @@ body{
 }
 
 .paperTitle{
-    font-weight: bold;
+     font-weight: 600;  /*semi-bold*/
```
Comment (vidhi-mody) [inline assets/css/research.css:231]: Please remove the extra whitespace.

File: assets/css/research.css [hunk 2]
```diff
@@ -177,6 +181,7 @@ body{
 .rConferences{
     font-style: italic;
     font-size: 0.88em;
+     font-weight: 400; /* regular */
```
Comment (vidhi-mody) [inline assets/css/research.css:184]: Please remove the extra whitespace.

**Review by vidhi-mody — CHANGES_REQUESTED** [review]

File: assets/css/research.css (hunk 1 above)
Comment (RitikPandey1) [reply assets/css/research.css]: done

File: assets/css/research.css (hunk 2 above)
Comment (RitikPandey1) [reply assets/css/research.css]: done

**Comment by smaranjitghose** [comment]
Use Poppins as the font please

**Comment by smaranjitghose** [comment]
While doing multiple changes that end up in a single PR, try to rebase them and squash the commits into a single commit. Makes the code more readable in long term!

**Comment by RitikPandey1** [comment]
> Use Poppins as the font please

done 👍

**Comment by RitikPandey1** [comment]
> While doing multiple changes that end up in a single PR, try to rebase them and squash the commits into a single commit. Makes the code more readable in long term!

I don't know how to rebase the commits ,could you please tell me or refer some article

**Comment by smaranjitghose** [comment]
[An article on Rebase](https://medium.com/osedea/git-rebase-powerful-command-507bbac4a234)
[Squash Commits](https://gist.github.com/patik/b8a9dc5cd356f9f6f980)

**Comment by RitikPandey1** [comment]
> [An article on Rebase](https://medium.com/osedea/git-rebase-powerful-command-507bbac4a234)
> [Squash Commits](https://gist.github.com/patik/b8a9dc5cd356f9f6f980)

Thanks ! 👍

**Comment by RitikPandey1** [comment]
> While doing multiple changes that end up in a single PR, try to rebase them and squash the commits into a single commit. Makes the code more readable in long term!

done

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Add bold font to buttons too.

File: assets/css/research.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*Import font*/
 
 @import url('https://fonts.googleapis.com/css2?family=Londrina+Outline&display=swap');
```
Comment (smaranjitghose) [inline assets/css/research.css:3]: I guess this is no longer required

Comment (RitikPandey1) [reply assets/css/research.css]: okay, I removed it

**Pull request merged**